src/style.css

```
@@ -0,0 +1,97 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+}
+
+.block_container {
+  display: flex;
+  flex-wrap: wrap;
+  flex-basis: 300px;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  width: 100%;
+  min-width: 300px;
+}
+
+.block_1 {
+  background-color: rgb(255,0,0);
+}
+
+.block_2 {
+  background-color: rgb(204,0,0);
+}
+
+.block_3 {
+  background-color: rgb(153,0,0);
+}
+
+.block_4 {
+  background-color: rgb(102,0,0);
+}
+
+.block_5 {
+  background-color: rgb(51,0,0);
+}
+
+.block_6 {
+  background-color: rgb(0,0,0);
+}
+
+@media screen and (min-width: 600px) {
```

screen is default mode so you can remove these redundancy
```suggestion
@media (min-width: 600px) {
```